cds_sorenson/api.py

```diff
@@ -34,12 +34,13 @@
 from flask import current_app
 
 from .error import InvalidAspectRatioError, InvalidResolutionError, \
-    SorensonError
+    SorensonError, TooHighResolutionError
 from .utils import _filepath_for_samba, generate_json_for_encoding, get_status
 
 
 def start_encoding(input_file, output_file, preset_quality,
-                   display_aspect_ratio, **kwargs):
+                   display_aspect_ratio, max_height=None, max_width=None,
```

Comment:
I'm not sure what's the best, but like this it's more explicit I think. And easy to document.
They are at the same relevance of `preset_quality` and `display_aspect_ratio`, both used to call the `get_preset_id` function as them.